spec/features/orders/creation_spec.rb

```diff
@@ -0,0 +1,111 @@
+RSpec.describe 'As a registered user', type: :feature do
+  describe "When I check out from my cart" do
+    before(:each) do
+      @user = create(:user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+
+      @mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      @meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+
+      @tire = @meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
+      @paper = @mike.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+      @pencil = @mike.items.create(name: "Yellow Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+
+      visit "/items/#{@paper.id}"
+      click_on "Add To Cart"
+      visit "/items/#{@paper.id}"
+      click_on "Add To Cart"
+      visit "/items/#{@tire.id}"
+      click_on "Add To Cart"
+      visit "/items/#{@pencil.id}"
+      click_on "Add To Cart"
+
+      visit "/cart"
+      click_on "Checkout"
+    end
+
+    it 'I can fill in my info, create a new order, am taken to orders page, see flash message, cart is empty' do
+      name = "Bert"
+      address = "123 Sesame St."
+      city = "NYC"
+      state = "New York"
+      zip = 10001
+
+      fill_in :name, with: name
+      fill_in :address, with: address
+      fill_in :city, with: city
+      fill_in :state, with: state
+      fill_in :zip, with: zip
+
+      click_button "Create Order"
+
+      new_order = Order.last
+
+      expect(current_path).to eq('/profile/orders')
+      expect(page).to have_content('Order ho! Ye successfully placed yer order. Make way in yer hold for loot!')
+      expect(page).to have_content(new_order.id)
+      expect(page).to have_content("Cart: 0")
+
+      # Old expectations of order show page:
```

Comment:
I think I did this on another branch, but for the sake of other folks: these expectations were left over from the original project but are not called for in our current user stories. I commented them out rather than deleting them because they might come in handy later if we reintroduce some of this functionality.